1-js/02-first-steps/01-hello-world/article.md

```diff
@@ -92,29 +91,29 @@ To attach several scripts, use multiple tags:
 ```
 
 ```smart
-As a rule, only the simplest scripts are put into HTML. More complex ones reside in separate files.
+Z reguły do kodu HTML wstawiane są bardzo proste skrypty, a bardziej złożone reguły trzyma się w osobnych plikach.
 
-The benefit of a separate file is that the browser will download it and store it in its [cache](https://en.wikipedia.org/wiki/Web_cache).
+Zaletą posiadania skryptów w osobnych plikach jest to, że przeglądarka go pobierze i zapisze w swojej pamięci [cache](https://en.wikipedia.org/wiki/Web_cache).
 
-Other pages that reference the same script will take it from the cache instead of downloading it, so the file is actually downloaded only once.
+Jeśli inne strony muszą skorzystać z tych skryptów, a plik został już pobrany, robią to z pamięci podręcznej, zamiast pobierać go ponownie.
 
-That reduces traffic and makes pages faster.
+To znacznie przyspiesza działanie strony i redukuje niepotrzebny ruch. 
 ```
 
-````warn header="If `src` is set, the script content is ignored."
-A single `<script>` tag can't have both the `src` attribute and code inside.
+````warn header="Jeśli atrybut `src` jest dodany, wtedy treść w tym tagu nie jest wykonywany."
```

Comment:
What do you think about: "Jeśli atrybut `src` jest dodany, wtedy skrypt nie jest wykonywany"?

Reply:
thanks